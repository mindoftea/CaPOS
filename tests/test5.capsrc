add 0x1F74CD, 0x0, m0
add 0x98601, 0x0, m1
jeq m0, 0x0, 0xb
jeq m1, 0x0, 0xb
jeq m0, m1, 0xb
add m0, 0x0, m3
add m1, 0x0, m0
add m3, 0x0, m1
sub m1, m0, m1
jgt m1, m0, 0x8
jlt 0x0, 0x1, 0x2
print m0, 0x0, 0x0
