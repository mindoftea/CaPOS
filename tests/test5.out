0x49e1
